src/utils/urlUtils.js

```diff
@@ -0,0 +1,125 @@
+export const requestURL = async (inputValue) => {
+  const BASE_URL = "http://localhost:3000";
+  const requestParam = {
+    url: inputValue,
+    wpm: Number(window.localStorage.getItem("wpm")),
+  };
+  const query = new URLSearchParams(requestParam).toString();
+  const path = `/articleSummary?${query}`;
+  const response = await fetch(BASE_URL + path);
+  const articleDatas = await response.json();
+
+  return articleDatas;
+};
+
+export const errorCase = (statusCode, message) => {
+  switch (statusCode) {
+    case 400:
+      return message[0].errorCode400;
+    case 512:
+      return message[0].errorCode512;
+    case 500:
+      return message[0].errorCode500;
+    default:
+      return true;
+  }
+};
+
+export const isValid = (
+  inputValue,
+  prevArticleDatas,
+  setMessageList,
+  textarea,
+  message,
+  link = null,
+) => {
+  const numOfURL = inputValue.split("http").length - 1;
+
+  if (numOfURL > 30) {
+    textarea.current.value = "";
+    textarea.current.style.height = "auto";
+
+    alert("URL 입력은 30개 이하로 해주세요");
+
+    return false;
+  }
+
+  if (inputValue.trim() === "") {
+    setMessageList((prev) => [
+      ...prev,
+      {
+        id: prev ? prev.length + 1 : 1,
+        message: message[0].onlySpace,
+        link: link || null,
```

Comment:
```suggestion
        link,
```

위(34번째 라인)에서 default 값(null)을 주고 있어서 생략할 수 있을 것 같아요.

Reply:
해당 줄 포함, 다른 로직들에도 반영했습니다.